SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialNetDriver.h

```diff
@@ -149,11 +149,17 @@ class SPATIALGDK_API USpatialNetDriver : public UIpNetDriver
 
 	void DelayedSendDeleteEntityRequest(Worker_EntityId EntityId, float Delay);
 
+	bool IsEntityListening(Worker_EntityId EntityId) const;
+	void RegisterListeningEntity(Worker_EntityId EntityId);
+	void UnregisterListeningEntity(Worker_EntityId EntityId);
```

Comment:
This doesn't seem to be called. I think you might want to call this when a client disconnects.

Reply:
Yes, I'll remove UnregisterListeningEntity.

Reply:
As said above, I think you might want to call this when a client disconnects. Otherwise that set will always grow with new players. This likely isn't going to be a problem before we replace events but it's still not idea.

Reply:
I will add the request to Unregister in `void USpatialReceiver::RemoveActor(Worker_EntityId EntityId)` for now.

Reply:
Added to RemoveEntity.